client/signup/utils.js

```diff
@@ -215,3 +216,18 @@ export function canResumeFlow( flowName, progress ) {
 	} );
 	return flowStepsInProgressStore.length > 0 && ! flow.disallowResume;
 }
+
+export const persistSignupDestination = url => {
+	const options = { path: '/' };
```

Comment:
I think we should also specify these options too:
- `expires`, because currently this is a session cookie which the browser throws away when it's closed. It feels like this should be persisted like the redux store.
- `sameSite: true`, can't think of a reason why we'd want another site snooping on this cookie value, so we should prevent it.

Reply:
Thanks for the suggestion, fixed in [4a1de1b](https://github.com/Automattic/wp-calypso/pull/34156/commits/4a1de1b704bc491522cee3315e37f18f8a31508f).